### PR TITLE
feat: improve norwegian translations

### PR DESCRIPTION
**Pull request by Johannes-Andersen**
Added a few missing Norwegian Bokmål translations that where missing

My first PR to the project, so let me know if I have done something stupid :)

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]
Checked the translations using google and looks good to me.

**Comment by ST-DDT** [comment]
Can you check why Github shows your commit as unverified?
grafik

**Comment by Johannes-Andersen** [comment]
> Can you check why Github shows your commit as unverified?
> grafik

Ah, I’ll get that fixed tomorrow. Didn’t have my hardware key with my GPG key plugged in so the commit did not get signed

**Comment by Johannes-Andersen** [comment]
> Can you check why Github shows your commit as unverified? grafik

Should be sorted now. All commits on the PR should be signed 👌🏻

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
> Should be sorted now. All commits on the PR should be signed 👌🏻

Works as expected. Thanks!

File: src/locales/nb_NO/color/human.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export default [
+  'beige',
+  'grønn',
+  'grå',
+  'gul',
+  'hvit',
+  'indigo',
+  'karmin',
+  'blå',
```
Comment (Shinigami92) [inline src/locales/nb_NO/color/human.ts:9]: Did we already defined somewhere that we want sorted arrays? Or will this be in a separated PR done via automation?
In both cases, we may also want a CI check to check if the arrays are sorted :thinking: ... could get complicated...

**Review by Shinigami92 — APPROVED** [review]

File: src/locales/nb_NO/color/human.ts (hunk 1 above)
Comment (xDivisionByZerox) [reply src/locales/nb_NO/color/human.ts:9]: We discussed that but wanted a way to automatically do this (like an eslint rule). So as long as this isn't in place it shouldn't be a requirement.

**Review by import-brain — APPROVED** [review]

**Pull request merged**